### PR TITLE
mobile version

### PR DESCRIPTION
**Pull request by latinogan**
### in this proyect we:

- add background images
- add linters configuration locally and ignore node modules
- add content and cards to the body

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -66,6 +69,177 @@ <h2 class="k">I´m Alex</h2>
...
+                 <img class="mob" src="pictures/circle.png" alt="rectangle">
+                 <h5 class="letrasp">Skills</h5>
+             </div>
+             <ul class="agrupador">
+                 <li>Codekit</li>
+                 <li>Github</li>
+                 <li>Codepen</li>
+                 <li>Terminal</li>
+             </ul>
+         </div>
```
Comment (Danieldotcomcoder) [inline index.html]: Languages, Frameworks, and Skills should be list items `li` as listed in the project requirement , (currently you have them as `h5` and `h3` tags) ,   Kindly fix this 😃✔

File: index.css [hunk 2]
```diff
@@ -112,19 +118,253 @@ nav {
...
+
+.about p {
+  width: 327px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  color: #f4f5f7;
+  margin-left: 20px;
+}
```
Comment (Danieldotcomcoder) [inline index.css]: Kindly increase the font size in the about section so it matches the Figma design  : 👍  😃👍

Your Design :
image

Figma Design : 
image

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @latinogan , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- No Linter Errors ✔
- Your project is very nice ✔
- Professional PR ✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.css (hunk 2 above)
Comment (latinogan) [reply index.css]: @[Danieldotcomcoder](https://github.com/Danieldotcomcoder) hello coach got a question the dimencion are375 x696
why it looks like that

Comment (latinogan) [reply index.css]: i was checking on figma and it shows the fond-size 16px and when I place the dimension 375 x695 I see it good

File: index.html (hunk 1 above)
Comment (latinogan) [reply index.html]: ill fixed thank you

File: index.css (hunk 2 above)
Comment (latinogan) [reply index.css]: im going to make the changes

**Review by JuliCarracedo — APPROVED** [review]
Hi @latinogan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**